### DISCUSSION TITLE
糾結要不要多花一年考研

### DISCUSSION THREAD

**Discussion by irenesunnn** [1101-GS4719-C/Python-Community-GS4719-C#3689] [General]
最近超級困擾要不要花一年準備考研。
說實話，我對目前的專業還蠻有興趣的，也想繼續深造。但是今年考研我覺得準備得不太充分，成績可能不太理想。一方面擔心考不上理想的學校，另一方面又怕浪費時間和家裡的錢。
我知道有些同學會選擇先工作幾年再考，但我怕工作後就更難靜下心來準備了。而且看到身邊的朋友都在為考研拼命，我自己也坐不住...
目前我的想法是：

如果今年成績真的不理想，就再花一年專心準備
但又擔心家人會覺得我在浪費時間
工作經驗確實重要，早點工作也許對未來發展更好？

大家有類似經歷嗎？真的很糾結要怎麼選擇...謝謝大家！

**Comment by Taoyi-Chiang** [comment]
我自己的經驗是如果讀研究所有猶豫其實還好，不妨換一個心態：考上了多一個選擇，沒考上也沒有損失。
早點工作當然可以卡位或累積實務經驗，當感到有所不足再回學校進修，確實是一個很好的選擇。
而且如果工作後就不會想認真準備，會不會表示其實自己也沒有真的這麼需要？
最後是家人的問題，如果會有家庭壓力的話，可能在某些情況下，分辨出是雙方之間的界線會是比較重要的事情。
畢竟家人沒辦法永遠幫自己負責，每一次選擇帶來的改變最後都只有自己可以承擔。
祝你能順利解決囉！
